src/main/java/uk/gov/hmcts/divorce/common/notification/AwaitingFinalOrderNotification.java

```diff
@@ -146,6 +147,8 @@ private Map<String, String> commonSolicitorTemplateVars(CaseData caseData, final
         templateVars.put(IS_DISSOLUTION, !caseData.isDivorce() ? YES : NO);
         templateVars.put(IS_SOLE, caseData.getApplicationType().isSole() ? YES : NO);
         templateVars.put(IS_JOINT, !caseData.getApplicationType().isSole() ? YES : NO);
+        templateVars.put(FINAL_ORDER_OVERDUE_DATE, caseData.getConditionalOrder().getGrantedDate().plusMonths(12)
+            .format(getDateTimeFormatterForPreferredLanguage(applicant.getLanguagePreference())));
```

Comment:
The date of issue uses DATE_TIME_FORMATTER, maybe we should use it here to make the dates consistent.

`.format(DATE_TIME_FORMATTER));`

Reply:
I think it might be more a case of updating the date of issue to use this here when the welsh ticket comes up